### PR TITLE
[PAPILIO-17] Update user model 

### PR DESCRIPTION
**Pull request by DrakeHunterK7**
# General info
Modified user model to include a new user attribute called "bio"

**Issue number**: PAPILIO-17

**Task description**: 
- Added "bio" attribute to User model
- Updated user schema to reflect this new "bio" attribute
- Updated user repo to make "bio" attribute part of the validation success

### CODE REVIEW

File: src/api/models/User.ts [hunk 1]
```diff
@@ -91,6 +92,9 @@ User.init(
                 isEmail: true
             },
             unique: true
+        },
+        bio: {
+            type: DataTypes.STRING
```
Comment (mth-1012) [inline src/api/models/User.ts:97]: You should either put a `defaultValue: 'something'` or `allowNull: false` option

File: src/api/schemas/user-schema.ts [hunk 2]
```diff
@@ -33,6 +33,8 @@ const countryCode = string({
     invalid_type_error: invalidMessage('Country Code', 'string')
 }).regex(new RegExp('^([0-9]|[1-9][0-9]|[1-9][0-9][0-9])$'), 'Invalid country code (0-999)');
 
+const bio = string({});
```
Comment (mth-1012) [inline src/api/schemas/user-schema.ts:36]: Add a tiny bit more details for `required_error` and `invalid_type_error`. All attributes follow the same structures!

**Review by mth-1012 — CHANGES_REQUESTED** [review]
After some tiny changes, we're good to go.

**Comment by DrakeHunterK7** [comment]
Acknowledged, making changes right now

File: src/api/schemas/user-schema.ts (hunk 2 above)
Comment (DrakeHunterK7) [reply src/api/schemas/user-schema.ts]: Done and done

**Comment by DrakeHunterK7** [comment]
Both comments addressed

**Review by mth-1012 — APPROVED** [review]

**Pull request merged**